src/sections/Initiatives.tsx

```diff
@@ -106,10 +106,12 @@ export const Initiatives = () => (
       </p>
       <p>
         <Button
-          aria-label="En savoir plus sur comment contribuer à La Suite numérique"
           href="/communs"
+          variant="outline"
+          size="large"
+          className="hover:bg-white"
```

Comment:
the size of the button were made smaller than on the design on purpose, after Johann received some feedbacks on the DSFR team; I think he forgot to update his Figma file.

Reply:
they are problematic on mobile, I am scared we should revert this commit ...

Reply:
So I keep everything as is except and just remove the `size` attribute so that they render smaller? Or is the issue also about the new style (outline style) and I just revert the [whole commit](https://github.com/numerique-gouv/lasuite-landingpage/pull/28/commits/d0761d54306c56f93974f8cf1550e63dddb47cc7)?

Reply:
Let's keep everything except the size, button should match DSFR height. Initially I picked their height from beta.gouv website, please feel free to update them to the relevant size, as soon as it's DSFR compliant 😇

Reply:
(the "DSFR compliant" is fuzzy, do what it takes to make it okayyy)